package.json

```
@@ -1,7 +1,17 @@
 {
   "name": "@iarna/toml",
   "version": "3.0.0",
-  "main": "toml.js",
+  "main": "./toml.js",
```

```suggestion
  "type": "commonjs",
  "main": "./toml.js",
```

This is the implicit default, but being explicit is the recommended approach:

> Package authors should include the "type" field, even in packages where all sources are CommonJS.

Ref.: https://nodejs.org/api/esm.html#esm_package_json_type_field

---

Thanks for the suggestions! This one seems great, I'll go ahead and do that.